examples/graph/plot_DAG_layouts.py

```diff
@@ -0,0 +1,63 @@
+"""
+======================
+DAG in Various Layouts
+======================
+Creates a Directed Acyclic Graph and shows the Graph in various layouts. 
+This can give an idea as to which layout can be chosen for these types of datasets.
+"""
+
+import networkx as nx
+import matplotlib.pyplot as plt
+
+
+G = nx.DiGraph()
+G.add_edges_from(
+    [
+        ("f", "a"),
+        ("a", "b"),
+        ("a", "e"),
+        ("b", "c"),
+        ("b", "d"),
+        ("d", "e"),
+        ("f", "c"),
+        ("f", "g"),
+        ("h", "f"),
+    ]
+)
```

Comment:
Just another little nit, but the DiGraph constructor accepts an edge list directly:

```suggestion
G = nx.DiGraph(
    [
        ("f", "a"),
        ("a", "b"),
        ("a", "e"),
        ("b", "c"),
        ("b", "d"),
        ("d", "e"),
        ("f", "c"),
        ("f", "g"),
        ("h", "f"),
    ]
)
```

Reply:
Yes understood! Committing the changes.